### DISCUSSION TITLE
Question 84: Cataloguing confiscated materials on EMu (from US Fish and Wildlife) [Herps]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#876] [Q&A] [answered]
Description: Amphibians & Reptiles Division has a bunch of items made from reptiles skins, turtle shells (?) like boots, bags/purse, briefcase, belts, jackets, snakes skins, python vertebrae necklace etc that I’ve just inventoried to put into Herps collections. I am guessing other divisions also get such confiscated items with/out documentation like permits. I’d like to catalog them (but not publicly visible). How would you recommend I catalog these on EMu.

**Comment by rondlg** [comment]
Response: 

- [LJ] Mammals might have accession information from Shedd in EMu already that they can share with Herps. Insects also have some processes in place for US Fish and Wildlife.

- [CM] Sounds like this was a big cross collection accession from Shedd - MaryAnn from Shedd is tracking down the paperwork for us. So there will be a single accession for it that all collections can attach to their catalogue records.

- If/When needed, how to document permits in EMu ([how-to doc here](https://github.com/fieldmuseum/EMu-Documentation/discussions/285))


**HOMEWORK** 
- Existing EMu records
1. [LH] to add the irn of the existing accession record to the github issue for everyone to use.

**Reply by LaurenEJohnson** [reply, marked as answer]
Hi all, The accession we created for this confiscated material is IRN 69833 (Accession Z-50673). We are still waiting for permits from Mary Anne Rogers - our point of contact at the Shedd. I'll remind her of this and see if she has managed to find anything.
